configure.in

```diff
@@ -554,11 +552,6 @@ if test "$GCC" = yes -a "$ICC" = no; then
   # remove this.
   PGAC_PROG_CC_CFLAGS_OPT([-Wno-unused-but-set-variable])
 
-  # We rely on /* fallthrough */ comments to signal explicit fallthrough, but
-  # some compilers (clang) don't recognize those and give spurious errors. Make
-  # sure the compiler supports fallthrough comments by explicitly requesting
-  # implicit-fallthrough level 3 (GCC's default).
-  PGAC_PROG_CC_CFLAGS_OPT([-Werror=implicit-fallthrough=3])
```

Comment:
-Wimplicit-fallthrough=3 and -Wvla are not included into -Wall (https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html). Let's add these flags near -Wall